### PR TITLE
feat: add script in precommit to prevent committing and pushing credentials

### PR DESCRIPTION
**Pull request by IyadhKhalfallah**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

**Review by joschkabraun — APPROVED** [review]
Nice! LGTM

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -15,9 +15,10 @@ repos:
     hooks:
       - id: isort
         exclude: ^now/thirdparty/
-  - repo: https://github.com/hadialqattan/pycln
-    rev: "v1.2.5"
+  - repo: local
     hooks:
-      - id: pycln
```
Comment (makram93) [inline .pre-commit-config.yaml:21]: Why are we removing this hook?

**Review by makram93 — CHANGES_REQUESTED** [review]
LGTM!
Any specific reason for removing pycln?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (IyadhKhalfallah) [reply .pre-commit-config.yaml:21]: Added back

**Review by makram93 — APPROVED** [review]

**Review by florian-hoenicke — APPROVED** [review]
LGTM

**Pull request merged**